backend/webserver/models.py

```diff
@@ -15,3 +16,61 @@ class FollowRequest(models.Model):
 class Follow(models.Model):
     follower = models.ForeignKey(Author, related_name='following_authors', on_delete=models.CASCADE)
     followee = models.ForeignKey(Author, related_name='followed_by_authors', on_delete=models.CASCADE)
+
+class Post(models.Model):
+
+    author = models.ForeignKey(Author,on_delete=models.CASCADE)
+
+    created_at = models.DateTimeField(verbose_name="date created",null=False, blank=False, default = datetime.now)
+
+    edited_at = models.DateTimeField("date edited")
+
+    title = models.CharField(max_length=200)
+
+    description = models.CharField(max_length=200)
+
+    source = models.CharField(max_length=200)
```

Comment:
```suggestion
    source = models.CharField(max_length=200, null=True)
```
- `null=True` because when the source of a post is not null, it could be a direct indicator that the post is a shared post.